MobilePlatformServices/Sources/MobilePlatformServices/App Info/AppInformationProvider.swift

```diff
@@ -28,7 +29,7 @@ public final class AppInformationService: AppInformationServicing {
     }
     
     public func fetchAppInfo() async throws -> App {
-        var request = URLRequest(url: AppEnvironment.appInfoURL)
```

Comment:
Should we remove this property from the `AppEnvironment` file now that we're not using it?